partiql-tests-data/eval/query/select/from-clause.ion

```diff
@@ -198,54 +198,70 @@ envs::{
   {
     name:"ordinalAccessWithNegativeIndex",
     statement:"SELECT temp[-2] FROM <<[1,2,3,4]>> AS temp",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:$bag::[
-        {
-        }
-      ]
-    }
+    assert:[
+      {
+        evalMode:EvalModeCoerce,
+        result:EvaluationSuccess,
+        output:$bag::[ {} ]
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail,
+      },
+    ]
   },
   {
     name:"ordinalAccessWithNegativeIndexAndBindings",
     statement:"SELECT temp[-2] FROM [[1,2,3,4]] AS temp",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:$bag::[
-        {
-        }
-      ]
-    }
+    assert:[
+      {
+        evalMode:EvalModeCoerce,
+        result:EvaluationSuccess,
+        output:$bag::[ {} ]
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail,
+      },
+    ]
   }
 ]
 
 'various types in from clause'::[
   {
     name:"rangeOverScalar",
     statement:"SELECT VALUE v FROM 1 AS v",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:$bag::[
-        1
-      ]
-    }
+    assert:[
+      {
+        evalMode:EvalModeCoerce,
+        result:EvaluationSuccess,
+        output:$bag::[ 1 ]
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail,
+      },
+    ]
   },
   {
     name:"rangeTwiceOverScalar",
     statement:"SELECT VALUE [v1, v2] FROM 1 AS v1, @v1 AS v2",
-    assert:{
-      evalMode:[EvalModeCoerce, EvalModeError],
-      result:EvaluationSuccess,
-      output:$bag::[
-        [
-          1,
-          1
+    assert:[
+      {
+        evalMode:EvalModeCoerce,
+        result:EvaluationSuccess,
+        output:$bag::[
+          [
+            1,
+            1
+          ]
         ]
-      ]
-    }
+      },
+      {
+        evalMode: EvalModeError,
+        result: EvaluationFail,
+      },
+    ]
```

Comment:
I believe the next test "rangeOverSexp" is also incorrect. An sexp is a sequence and therefore ordered. The result should be `[a, b, c ]` not a singleton bag.

Notice that we turn an input Ion list into a collection that we range over, but here we treat the sexp collection as a single value. This does not make sense.

Reply:
Added! Also found and fixed some more incorrect tests -- see 1da4e8b89e9f50072e9540d1bd6e51fd0a992b25.